docs/Formula-Cookbook.md

```diff
@@ -752,7 +752,7 @@ The symlinks created by [`install_symlink`](https://rubydoc.brew.sh/Pathname#ins
 
 ### Rewriting a script shebang
 
-Some formulae install executable scripts written in an interpreted language such as Python or Perl. Homebrew provides a `rewrite_shebang` method to rewrite the shebang of a script. This replaces a script's original interpreter path with the one installed by Homebrew and guarantees that the correct interpreter is used at execution time.
+Some formulae install executable scripts written in an interpreted language such as Python or Perl. Homebrew provides a `rewrite_shebang` method to rewrite the shebang of a script. This replaces a script's original interpreter path with the one the formula depends on. This guarantees that the correct interpreter is used at execution time. It is only needed when the build system doesn't already do it (e.g., often when using `pip` or Perl `ExtUtils::MakeMaker`).
```

Comment:
```suggestion
Some formulae install executable scripts written in an interpreted language such as Python or Perl. Homebrew provides a `rewrite_shebang` method to rewrite the shebang of a script. This replaces a script's original interpreter path with the one the formula depends on. This guarantees that the correct interpreter is used at execution time. This isn't required if the build system already handles it (e.g. often with `pip` or Perl `ExtUtils::MakeMaker`).
```

Right now I think it's a little unclear whether `pip` and `ExtUtils::Maker` _do_ or _don't_ do it automatically.